examples/mysql/README.md

```diff
@@ -140,7 +137,7 @@ Let's say someone accidentally deleted the test database using the following com
 # Connect to MySQL by running a shell inside MySQL's pod
 $ kubectl exec -ti $(kubectl get pods -n mysql-test --selector=app.kubernetes.io/instance=mysql-release -o=jsonpath='{.items[0].metadata.name}') -n mysql-test -- bash
 
-$ mysql --user=root --password=asd#45@mysqlEXAMPLE
+$ mysql --user=root --password=<mysql-root-password>
```

Comment:
should we change this at other places as well?

Reply:
there is only 1 place where we have hard-coded this but it's in the example command. so I think it's okay to keep it.